coordinator/src/routes.rs

```diff
@@ -57,6 +57,7 @@ pub fn router(node: Node, pool: Pool<ConnectionManager<PgConnection>>) -> Router
         )
         .route("/api/orderbook/websocket", get(websocket_handler))
         .route("/api/trade", post(post_trade))
+        .route("/api/log_channel_list", get(log_channel_list))
```

Comment:
Love this endpoint where you call something and seemingly nothing happens 😅

You could copy pasa the code from our PoC where we manually serialize the channel details

Reply:
Hahaha, I will do that then.

Reply:
See https://github.com/get10101/10101/pull/271/commits/f548e95d9831776022b41da42727c5a4ba30e565, https://github.com/get10101/10101/pull/271/commits/0701aa4ac39ab8749980c4f6442a1498f4979f53, https://github.com/get10101/10101/pull/271/commits/34f880c42bfbec767617415a9c9bac3d17bc05a4 and https://github.com/get10101/10101/pull/271/commits/4eb2f2b291c274dac5ee8e5dfbaef44f87354821.